components/renku_pack_builder/manifests/buildstrategy.yaml

```
@@ -0,0 +1,69 @@
+---
+apiVersion: shipwright.io/v1beta1
+kind: BuildStrategy
+metadata:
+  name: renku-buildpacks
+spec:
+  parameters:
+    - name: platform-api-version
+      description: The referenced version is the minimum version that all relevant buildpack implementations support.
+      default: "0.12"
+    - name: process-type
+      description: |
+        The process to execute at launch time
+        https://buildpacks.io/docs/for-buildpack-authors/how-to/write-buildpacks/specify-launch-processes/
+      default: "web"
```

Since we add `bash` with the `procfile` step, we should default to it.

```suggestion
      default: "bash"
```